assets/xml/objects/object_eg.xml

```diff
@@ -1,106 +1,140 @@
+        <Limb Name="gEyegoreEyeColliderLimb" Type="Standard" EnumName="EYEGORE_LIMB_EYE_COLLIDER" Offset="0x9590" /> 
+        <Limb Name="gEyegoreEyeballLimb" Type="Standard" EnumName="EYEGORE_LIMB_EYEBALL" Offset="0x959C" /> 
+        <Limb Name="gEyegoreHipsLimb" Type="Standard" EnumName="EYEGORE_LIMB_HIPS" Offset="0x95A8" /> 
+        <Limb Name="gEyegoreLeftThighLimb" Type="Standard" EnumName="EYEGORE_LIMB_LEFT_THIGH" Offset="0x95B4" /> 
+        <Limb Name="gEyegoreLeftShinLimb" Type="Standard" EnumName="EYEGORE_LIMB_LEFT_SHIN" Offset="0x95C0" /> 
+        <Limb Name="gEyegoreLeftFootLimb" Type="Standard" EnumName="EYEGORE_LIMB_LEFT_FOOT" Offset="0x95CC" /> 
+        <Limb Name="gEyegoreRightThighLimb" Type="Standard" EnumName="EYEGORE_LIMB_RIGHT_THIGH" Offset="0x95D8" /> 
+        <Limb Name="gEyegoreRightShinLimb" Type="Standard" EnumName="EYEGORE_LIMB_RIGHT_SHIN" Offset="0x95E4" /> 
+        <Limb Name="gEyegoreRightFootLimb" Type="Standard" EnumName="EYEGORE_LIMB_RIGHT_FOOT" Offset="0x95F0" /> 
+        <Limb Name="gEyegoreFauldsLimb" Type="Standard" EnumName="EYEGORE_LIMB_FAULDS" Offset="0x95FC" /> 
+
+        <Skeleton Name="gEyegoreSkeleton" Type="Flex" LimbType="Standard" LimbNone="EYEGORE_LIMB_NONE" LimbMax="EYEGORE_LIMB_MAX" EnumName="EyegoreLimbs" Offset="0x9664" />
+
+        <Animation Name="gEyegoreSlamWaitAnim" Offset="0x9D2C" />
+        <Animation Name="gEyegoreSlamEndAnim" Offset="0xA204" />
+        <Animation Name="gEyegoreLaserAnim" Offset="0xA484" />
+        <Animation Name="gEyegoreUnusedLaserEndAnim" Offset="0xA73C" />
+        <Animation Name="gEyegoreDamagedAnim" Offset="0xADF8" />
+        <Animation Name="gEyegoreDeathAnim" Offset="0xBC60" />
+        <Animation Name="gEyegoreRetreatAnim" Offset="0xBEB8" />
+        <Animation Name="gEyegoreLeftPunchAnim" Offset="0xC434" />
+        <Animation Name="gEyegoreRightPunchAnim" Offset="0xC9B4" />
+        <Animation Name="gEyegoreStunnedAnim" Offset="0xCC4C" />
+        <Animation Name="gEyegoreStunEndAnim" Offset="0xCE54" />
+        <Animation Name="gEyegoreSitAnim" Offset="0xD3D8" />
+
+        <DList Name="gEyegoreUnusedStoneDL" Offset="0xDCC0" />
+        <Texture Name="gEyegoreUnusedStoneTex" OutName="eyegore_unused_stone" Format="i4" Width="64" Height="64" Offset="0xDEA8" />
+
+        <Animation Name="gEyegoreUnusedChaseAnim" Offset="0xEA10" />
+        <Animation Name="gEyegoreStandAnim" Offset="0xEE4C" />
+        <Animation Name="gEyegoreUnusedWalkAnim" Offset="0xF524" />
+        <Animation Name="gEyegoreUnusedIdleAnim" Offset="0xFB0C" />
+        <Animation Name="gEyegoreWalkAnim" Offset="0x100C0" />
+
+        <DList Name="gEyegoreStoneDL" Offset="0x10240" />
+        <Texture Name="gEyegoreStoneTex" OutName="eyegore_stone" Format="rgba16" Width="36" Height="36" Offset="0x10310" />
+
+        <Texture Name="gEyegoreDuplicateEffectImpactTex" OutName="eyegore_duplicate_effect_impact" Format="ia8" Width="64" Height="64" Offset="0x10D30" />
+
```

Comment:
I don't think you need all the spaces between this texture and the ones surrounding it.